moveit_ros/planning/trajectory_execution_manager/src/trajectory_execution_manager.cpp

```diff
@@ -51,6 +52,7 @@ static const double DEFAULT_CONTROLLER_GOAL_DURATION_MARGIN = 0.5;  // allow 0.5
                                                                     // after scaling)
 static const double DEFAULT_CONTROLLER_GOAL_DURATION_SCALING =
     1.1;  // allow the execution of a trajectory to take more time than expected (scaled by a value > 1)
+static const bool DEFAULT_CONTROL_MULTI_DOF_JOINT_VARIABLES = false;
```

Comment:
Do we need this as a static const variable, can't we treat it like the default value of `wait_for_trajectory_completion_`?

Reply:
@henningkayser Why do we need this parameter? Can control multi-DOF joints be true by default since this is the only way to get multi-DFO execution working?

Reply:
Because by default multi-dof trajectories should be controlled as multi-dof trajectories :man_shrugging: . Converting to joint variables is a feature that we use in a specific use case with ros2_control, so I would not set this as new default.